frontend/simple/views/components/Voting/Voting.vue

```diff
@@ -1,47 +1,50 @@
 <template>
-  <div class="is-flex gi-voting">
+<transition name="gi-fade">
```

Comment:
What is this file? `Voting.vue` is very unclear to me as to its purpose. The only way I could figure it out is by running the program. Can we rename it to something clearer?

Reply:
Indeed it was hard for me to find a name... What name do you suggest?

Reply:
Maybe `VoteProposalBox.vue` or something like that?

Reply:
Perhaps even better, just: `ProposalBox.vue`